### PR TITLE
[CWS] introduce SECL operator override

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR introduce a way to override default behavior of SECL operator per field. This aims to be used by the following PR addressing symlink resolution issue. This goal will be to transform : 

```
open.file.path == "/usr/bin/python3"

into 

open.file.path in ["/usr/bin/python3", "/usr/bin/python3.9"]
```

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by lebauce — APPROVED** [review]

File: cmd/agent/android/.settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
 connection.project.dir=
 eclipse.preferences.version=1
+gradle.user.home=
+java.home=/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
+jvm.arguments=
+offline.mode=false
+override.workspace.settings=true
+show.console.view=true
+show.executions.view=true
```
Comment (vickenty) [inline cmd/agent/android/.settings/org.eclipse.buildship.core.prefs:13]: Is this change (and one above) intentional? They do not seem related to this PR.

Comment (safchain) [reply cmd/agent/android/.settings/org.eclipse.buildship.core.prefs:13]: oups, will remove it

File: cmd/agent/android/.project [hunk 2]
```diff
@@ -14,4 +14,15 @@
...
+		<filter>
+			<id>1641226798995</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
```
Comment (djmitche) [inline cmd/agent/android/.project:27]: This change, too.
